src/environments/environment.prod.ts

```diff
@@ -9,4 +9,5 @@ export const environment: EnvironmentConfig = {
     redirectUri: `${window.location.origin}/ogloszenia/`,
     logoutUrl: `${window.location.origin}/ogloszenia/`,
   },
+  applicationInsightsConnectionString: '',
```

Comment:
Is an empty string going to work here?

Reply:
No it's not. Gonna fix this one as well.

Reply:
@luziczek can you explain how that works? The docs say:
> 1. First you need an Application Insights resource. If you don't already have a resource and instrumentation key, follow the [create a new resource instructions](https://docs.microsoft.com/en-us/azure/azure-monitor/app/create-new-resource).
> 2. Copy the instrumentation key (also known as "iKey") or [connection string](https://docs.microsoft.com/en-us/azure/azure-monitor/app/javascript#connection-string-setup) for the resource where you want your JavaScript telemetry to be sent (from step 1.) You'll add it to the instrumentationKey or connectionString setting of the Application Insights JavaScript SDK.

which suggests we need to copy this string from somewhere.

Reply:
@sfc-gh-pufnalewski is gonna provide connectionString and since it's a sensitive content it shouldn't be in the repo. I assume it will be added on build phase but empty string here is not needed at the moment and it should be reflected as optional in the model (this change is coming as well).

Reply:
Now, the question is how to integrate passing that string with the Angular build. :) I expect some changes to this repo are also necessary to make it happen.

Reply:
OK, maybe this will be written to the `environment.prod.ts` file during the build so we're good.